source/hic_hal/target_config.h

```diff
@@ -40,6 +40,9 @@ extern "C" {
 // This can vary from target to target and should be in the structure or flash blob
 #define TARGET_AUTO_INCREMENT_PAGE_SIZE    (1024)
 
+//Additional flash and ram regions
+#define MAX_ADD_FLASH_REGION                3
+#define MAX_ADD_RAM_REGION                  3
 /**
```

Comment:
Can you please insert a blank line here between the #define and start of the comment?